pkg/background/process_funding_schedule.go

```diff
@@ -254,18 +255,36 @@ func (p *ProcessFundingScheduleJob) Run(ctx context.Context) error {
 					continue
 				}
 
+				spendingFunding, err := p.repo.GetSpendingFunding(span.Context(), spending.BankAccountId, spending.SpendingId)
```

Comment:
This might result in some doubled up queries with `GetSpendingByFundingSchedule` where we hit the spending funding table twice?